### ISSUE TITLE
[Bug]: mo hang occurs because of multi-tenant load

### ISSUE THREAD

**Issue by tianyahui-python** [matrixorigin/matrixone#9992]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):e930b2bb133017738510c86054604b518b415ddb
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

Reproducible Steps:
sys tenant execute:
```
mysql> create account test_tenant_1 admin_name 'test_account' identified by '111';
Query OK, 0 rows affected (0.45 sec)

mysql> create account test_tenant_2 admin_name 'test_account' identified by '111';
Query OK, 0 rows affected (0.56 sec)
```

test_tenant_1 execute：
```
create databases db1;
use db1;
drop table if exists t1;
create table t1(
clo1 tinyint,
clo2 smallint,
clo3 int,
clo4 bigint,
clo5 tinyint unsigned,
clo6 smallint unsigned,
clo7 int unsigned,
clo8 bigint unsigned,
col9 float,
col10 double,
col11 varchar(255),
col12 Date,
col13 DateTime,
col14 timestamp,
col15 bool,
col16 decimal(5,2),
col17 text,
col18 json,
col19 blob,
col20 binary(255),
col21 varbinary(255)
);

 LOAD DATA URL s3option{"endpoint"='s3.us-west-2.amazonaws.com', "access_key_id"='xxx', "secret_access_key"='xxx', "bucket"='mo-load-data', "filepath"='/new_20_columns/10w_20_columns_load_data.csv', "region"='us-west-2', "compression"='none'} INTO TABLE t1 FIELDS TERMINATED BY ',' ENCLOSED BY '\"' LINES TERMINATED BY '\n' parallel 'true';
```

test_tenant_2 execute:
```
create databases db1;
use db1;
drop table if exists t1;
create table t1(
clo1 tinyint,
clo2 smallint,
clo3 int,
clo4 bigint,
clo5 tinyint unsigned,
clo6 smallint unsigned,
clo7 int unsigned,
clo8 bigint unsigned,
col9 float,
col10 double,
col11 varchar(255),
col12 Date,
col13 DateTime,
col14 timestamp,
col15 bool,
col16 decimal(5,2),
col17 text,
col18 json,
col19 blob,
col20 binary(255),
col21 varbinary(255)
);

 LOAD DATA URL s3option{"endpoint"='s3.us-west-2.amazonaws.com', "access_key_id"='xxx', "secret_access_key"='xxx', "bucket"='mo-load-data', "filepath"='/new_20_columns/10w_20_columns_load_data.csv', "region"='us-west-2', "compression"='none'} INTO TABLE t1 FIELDS TERMINATED BY ',' ENCLOSED BY '\"' LINES TERMINATED BY '\n' parallel 'true';
```
test_tenant_1 和 test_tenant_2 同时执行
current execute result： mo cannot execute sql, hang

**Comment by tianyahui-python** [comment]
Single tenants have the same problem.

**Comment by guguducken** [comment]
no progress

**Comment by zhangxu19830126** [comment]
fixed by #10001

**Comment by tianyahui-python** [comment]
已通过本地测试
